### PR TITLE
21 admin dashboard statistics top customers

### PR DESCRIPTION
**Pull request by sjcowans**

### CODE REVIEW

File: spec/features/admin/index_spec.rb [hunk 1]
```diff
@@ -3,55 +3,84 @@
 RSpec.describe 'admin_dashboard', type: :feature do
   describe 'As an admin, when I visit the admin dashboard' do
     before(:each) do
-      @customer_1 = create(:customer)
```
Comment (margdaly) [inline spec/features/admin/index_spec.rb:6]: Good Call removing these!! I forgot to do this early! 
My tone on factory/faker have changed a bit heheh

File: app/models/customer.rb [hunk 2]
```diff
@@ -1,4 +1,12 @@
 class Customer < ApplicationRecord
   has_many :invoices
   has_many :transactions, through: :invoices
+
+  def self.top_customers
```
Comment (margdaly) [inline app/models/customer.rb:5]: Do we have a test for this in models/customer_spec.rb?

or did we put it in the merchant_spec.rb?

**Review by margdaly — CHANGES_REQUESTED** [review]
Does the Customer.top_5 mean we ought to reomve the method in the Merchants model? the   top_5_customers? and move the test into the spec/customer_spec.rb?

**Review by NickolasAM — APPROVED** [review]
Looks great!

File: app/models/customer.rb (hunk 2 above)
Comment (sjcowans) [reply app/models/customer.rb:5]: Adding now

**Review by NickolasAM — APPROVED** [review]
Looks great again!

**Review by margdaly — APPROVED** [review]
Awesome!! Thank you!!

**Pull request merged**